articles/create-login-and-register-system-with-C#-winforms/index.md

```diff
@@ -0,0 +1,430 @@
+### Introduction
+
+C# or C-sharp is an object-oriented programming language developed by Microsoft that runs on a framework called .Net framework. The language has been used to create many applications, including web applications, mobile applications, computer games, desktop applications, database applications, virtual reality applications, etc.
+
+### Prerequisites
+
+* A code editor a good example is [Visual Studio](https://visualstudio.microsoft.com/vs/).
+* Knowledge of C#, SQL.
+* Running relational database management in this case, we will use [MySQL](https://www.mysql.com/).
+* Visual Studio Knowledge.
+
+### Step I: Create a database and table with required columns.
+
+``` sql
+
+Create database userdata;
+```
+
+![DATABASE CREATION]((/engineering-education/create-login-and-register-system-with-C#-winforms/create-db-cmd.png))
```

Comment:
This image URL is enclosed in double round brackets....modify where applicable throughout this article.